### ISSUE TITLE
Rearrange seams to add CPU-based neural network random number generator and S-GraTe Pretransformer

### ISSUE THREAD

**Issue by AStarCale** [AStarStarship/Script2#330]
## Problem

The problem is that random numbers take a really long time to generate, and there is a smarter way to sort tokens for LLM.

## Solution

The solution is to use a CPU based neural network to generate random numbers using a running XOR after every floating-point division instruction. By the original design of the API as a microframework to build other frameworks from a neural network API in software didn't make sense. This is no longer the case. The Chinese Room would greatly benefit from the S-GraTe (Spectral Graph Token) Pretransformer. It adds the missing math functionality, and we need Math algorisms specifically optimized for specific number ranges for NN input, specifically to calculate softmax for neural network inputs in the S-GraTe space.

### File Affected

1. `?`

## Hierarchy

* #11

## Tags

Mission

### Sessions

* AStarCale/.github#9

## License

Copyright [AStarship™](https://astarship.net).
